functions/Get-DbaDatabase.ps1

```diff
@@ -99,8 +99,7 @@ Returns databases on multiple instances piped into the function
         [parameter(ParameterSetName = "Encrypted")]
         [switch]$Encrypted,
         [parameter(ParameterSetName = "Status")]
-        [ValidateSet('EmergencyMode', 'Normal', 'Offline', 'Recovering', 'Restoring', 'Standby', 'Suspect')]
-        [string]$Status,
```

Comment:
Apologies, something strange was going on in my session. That error was what I was getting with ValidateSet

Bounced box and it works now. Will push up